static/app/views/organizationGroupDetails/grouping/groupedIssues/index.tsx

```diff
@@ -0,0 +1,151 @@
+import {Fragment, useState} from 'react';
+import styled from '@emotion/styled';
+
+import GroupingActions from 'app/actions/groupingActions';
+import Alert from 'app/components/alert';
+import Button from 'app/components/button';
+import Confirm from 'app/components/confirm';
+import {Panel} from 'app/components/panels';
+import QuestionTooltip from 'app/components/questionTooltip';
+import {t, tn} from 'app/locale';
+import {Fingerprint} from 'app/stores/groupingStore';
+import space from 'app/styles/space';
+import {Group} from 'app/types';
+
+import HeaderTitle from '../headerTitle';
+
+import GroupingCard from './groupingCard';
+
+type Props = {
+  groupId: Group['id'];
+  issues: Fingerprint[];
+};
+
+function GroupedIssues({issues, groupId}: Props) {
+  if (!issues.length) {
+    return null;
+  }
+
+  const [activeGrouping, setActiveGrouping] = useState(0);
+
+  const groupings = [
+    [
+      {
+        childId: issues[0].id,
+        childLabel: issues[0].label,
+        eventCount: issues[0].eventCount,
+        lastSeen: issues[0].lastSeen,
+        latestEvent: issues[0].latestEvent,
+      },
+    ],
+    issues[0].children,
+  ];
+
+  const regroupDisabled = issues.length !== 1;
+
+  function handleSplit() {
+    GroupingActions.toggleUnmerge([issues[0].id, issues[0].latestEvent.id]);
```

Comment:
NIT: lots of `issues[0].`, should we destructure?

Reply:
this will be removed soon 😉